Makefile

```
@@ -13,6 +13,12 @@ geth:
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/geth\" to launch geth."
 
+suave:
+	$(GORUN) build/ci.go install ./cmd/geth
+	cd $(GOBIN) && ln -s geth suave
```

```suggestion
	cd $(GOBIN) && rm -f suave && ln -s geth suave
```

need to remove first, otherwise getting this error on repeated executions:

```
cd ./build/bin && ln -s geth suave
ln: failed to create symbolic link 'suave': File exists
make: *** [suave] Error 1
```

---

Thanks for catching - have attended to both comments here :pray: